### PR TITLE
Added MANIFEST.in to include docs, tests and tools in source package.

### PR DESCRIPTION
**Pull request by anandology**
Issue #627

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,3 @@
+include requirements.txt
+include test_requirements.txt
+recursive-include docs tests toolst
```
Comment (iredmail) [inline MANIFEST.in]: Typo: `toolst` -> `tools`

**Comment by iredmail** [comment]
Also, we need to document the commands used to package web.py release somewhere, so that new/other maintainers won't make same mistake like me.

File: MANIFEST.in (hunk 1 above)
Comment (anandology) [reply MANIFEST.in]: Fixed.

**Comment by anandology** [comment]
> Also, we need to document the commands used to package web.py release somewhere, so that new/other maintainers won't make same mistake like me.

Agreed, created an issue for this. #629

**Comment by iredmail** [comment]
@anandology Would you like to finish this PR?

**Pull request merged**